clang/lib/CIR/CodeGen/CIRGenExpr.cpp

```diff
@@ -165,6 +165,34 @@ LValue CIRGenFunction::emitDeclRefLValue(const DeclRefExpr *e) {
   return LValue();
 }
 
+/// Emit code to compute the specified expression which
+/// can have any type.  The result is returned as an RValue struct.
+RValue CIRGenFunction::emitAnyExpr(const Expr *e, bool ignoreResult) {
```

Comment:
is `ignoreResult` used anywhere?    Might need a cast-to-void here to prevent diagnostics.